src/test/crimson/test_errorator.cc

```diff
@@ -18,6 +18,23 @@ struct errorator_test_t : public seastar_test_suite_t {
       }
     });
   }
+  struct noncopyable_t {
```

Comment:
How about [`boost::noncopyable`](https://www.boost.org/doc/libs/1_65_0/libs/core/doc/html/core/noncopyable.html)?

Reply:
actually, i was using `boost::noncopyable`, but after a second thought, still think that the homebrew version is more straightforward.